### DISCUSSION TITLE
[Feature] Search in person change name dialog

### DISCUSSION THREAD

**Discussion by Tyree** [immich-app/immich#5206] [Feature Request]
### The feature

It would be a big time saver if naming people from the people page gave a fuzzy search similar to how it works on the individual person pages. Type a few letters, choose an existing name, and move on.
As of now, you have to type the whole name, click OK, then it will pop up with a name that matches.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5209

**Reply by Tyree** [reply]
#5209 is not the same thing (although also nice - but I use CTRL+F5 to search the name anyway).
What I'm suggesting is the user never has to leave the People page. When they click the dot menu on a person and click Change name, the change name blank should act as it does at the top of the person page - auto-suggesting possible matches as the user types. Big time saver.
